lib/fastlane/actions/actions_helper.rb

```diff
@@ -18,7 +18,7 @@ def self.lane_context
     # Pass a block which should be tracked. One block = one testcase
     # @param step_name (String) the name of the currently built code (e.g. snapshot, sigh, ...)
     def self.execute_action(step_name)
-      raise 'No block given'.red unless block_given?
+      fail 'No block given'.red unless block_given?
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.